### PR TITLE
Updates for Python SDK

### PR DESCRIPTION
**Pull request by phillipgr4vy**
Updates for Python SDK

### CODE REVIEW

File: docs/web-checkout/sdks.mdx [hunk 1]
```diff
@@ -148,6 +157,14 @@ instance. Refer to the integrations panel of your dashboard for more details.
 
 :::
 
+  </TabItem>
+  <TabItem value="python">
+
+```python
+from gr4vy_python import Gr4vyClient
```
Comment (steve-gr4vy) [inline docs/web-checkout/sdks.mdx:164]: Should now be `from gr4vy import ..`, right?

**Review by steve-gr4vy — COMMENTED** [review]
Just the one comment regarding renaming gr4vy_python to `gr4vy`

**Review by steve-gr4vy — APPROVED** [review]
Looks good.

**Comment by phillipgr4vy** [comment]
Updates for Python SDK documentation

**Pull request merged**